src/main/java/nextstep/security/filter/converter/LoginFormAuthenticationConverter.java

```diff
@@ -0,0 +1,27 @@
+package nextstep.security.filter.converter;
+
+import jakarta.servlet.http.HttpServletRequest;
+import nextstep.security.authentication.Authentication;
+import nextstep.security.authentication.UsernamePasswordAuthenticationToken;
+
+import java.util.Map;
+
+public class LoginFormAuthenticationConverter implements AuthenticationConverter {
+
+    @Override
+    public Authentication convert(final HttpServletRequest request) {
+        final Map<String, String[]> parameterMap = request.getParameterMap();
+
+        String username;
+        String password;
+
+        try {
+            username = parameterMap.get("username")[0];
+            password = parameterMap.get("password")[0];
+        } catch (RuntimeException e) {
+            throw new IllegalArgumentException(e);
+        }
+
+        return new UsernamePasswordAuthenticationToken(username, password);
```

Comment:
```suggestion
        try {
            String username = parameterMap.get("username")[0];
            String password = parameterMap.get("password")[0];
            return new UsernamePasswordAuthenticationToken(username, password); 
        } catch (RuntimeException e) {
            throw new IllegalArgumentException(e);
        }
```
가 조금 더 변수를 불변하게 사용하는 안정적인 코드가 될 수 있겠네요 :)